### PR TITLE
Add missing printing methods and fix a binding reference bug.

### PR DESCRIPTION
**Pull request by LucasJSch**
This PR adds some printing functionalities to `InsectType` and `CellMap` to be used in future PRs.

Also, it fixes a small binding reference in the `Environment` class.

### CODE REVIEW

File: course/include/insects_metrics.h [hunk 1]
```diff
@@ -11,6 +13,27 @@ namespace simulation {
 class Insect;
 enum class InsectType {Undefined, Ant, Doodlebug};
 
+namespace {
+std::string InsectTypeToString(InsectType type) {
+    if (type == InsectType::Ant) {
+        return std::string("Ant");
+    }
+    if (type == InsectType::Doodlebug) {
```
Comment (ernestmc) [inline course/include/insects_metrics.h]: This can be done with a `switch` statement

File: course/include/insects_metrics.h [hunk 2]
```diff
@@ -11,6 +13,27 @@ namespace simulation {
...
+        return std::string("Doodlebug");
+        }
```
Comment (ernestmc) [inline course/include/insects_metrics.h]: indent

File: course/include/insects_metrics.h [hunk 3]
```diff
@@ -11,6 +13,27 @@ namespace simulation {
...
+    return std::string("Undefined");
+    }
+
+char InsectTypeToChar(InsectType type) {
+    if (type == InsectType::Ant) {
+        return 'A';
+    }
+    if (type == InsectType::Doodlebug) {
```
Comment (ernestmc) [inline course/include/insects_metrics.h]: Can use `switch`

File: course/test/src/Environment_TEST.cc [hunk 4]
```diff
@@ -64,7 +64,12 @@ GTEST_TEST(EnvironmentTest, SimulationFinishedWhenThereAreOnlyAnts) {
 GTEST_TEST(EnvironmentTest, SimulationFinishedWhenTheDoodlebugEatsTheAntAndThereAreNoMoreInsects) {
     Environment env(1, 2, 1, 1);
     EXPECT_FALSE(env.SimulationFinished());
-    env.RunRound();
+    std::vector<InsectsRoundMetrics> metrics = env.RunRound();
+    for (const InsectsRoundMetrics metric : metrics ) {
+        if (metric.GetInsectType() == InsectType::Ant) {
+            EXPECT_EQ(metric.GetDeadCount(), (unsigned)1);
```
Comment (ernestmc) [inline course/test/src/Environment_TEST.cc]:
```suggestion
            EXPECT_EQ(metric.GetDeadCount(), 1u);
```

**Review by ernestmc — APPROVED** [review]
lgtm

File: course/include/insects_metrics.h (hunk 1 above)
Comment (LucasJSch) [reply course/include/insects_metrics.h]: Done.

File: course/include/insects_metrics.h (hunk 2 above)
Comment (LucasJSch) [reply course/include/insects_metrics.h]: Done.

File: course/include/insects_metrics.h (hunk 3 above)
Comment (LucasJSch) [reply course/include/insects_metrics.h]: Done.

File: course/test/src/Environment_TEST.cc (hunk 4 above)
Comment (LucasJSch) [reply course/test/src/Environment_TEST.cc]: Nice one :+1:
Done

**Pull request merged**